toolchain/check/context.cpp

```diff
@@ -61,10 +61,8 @@ auto Context::VerifyOnFinish() -> void {
 }
 
 auto Context::AddNode(SemIR::Node node) -> SemIR::NodeId {
-  auto node_id = semantics_ir_->AddNodeInNoBlock(node);
-  node_block_stack_.AddNodeId(node_id);
-  CARBON_VLOG() << "AddNode " << node_block_stack_.Peek() << ": " << node
-                << "\n";
+  auto node_id = node_block_stack_.AddNode(node);
+  CARBON_VLOG() << "AddNode: " << node << "\n";
```

Comment:
You might consider moving this VLOG to the NodeBlockStack::AddNode, and removing this wrapper. Since node_block_stack is visible for callers, and NodeBlockStack can VLOG, nothing would really be lost and it'd remove some indirection.

But I'm going to go ahead and approve since I don't think this suggestion needs to be done in this PR.